contents/blog/best-open-source-feature-flag-tools.md

```diff
@@ -32,54 +31,51 @@ In this guide, we’ll explore some of the best open source feature flag tools,
 
 If you want to run experiments as well, we recommend reading our guide to the [best open-source A/B testing tools](/blog/best-open-source-ab-testing-tools).
 
-## 1. [PostHog](https://posthog.com)
+## 1. PostHog
 
 ![PostHog](../images/blog/open-source-feature-flags/posthog-feature-flags.png)
 
-PostHog is an all-in-one [Product OS](https://posthog.com/product) that captures and visualizes all of the data your team needs in one place. Complete with product analytics, feature flags, session recording, heatmaps, funnels and more, it’s a great choice if you want to build a comprehensive, data-driven product strategy, fast.
+[PostHog](https://posthog.com) is an all-in-one [Product OS](https://posthog.com/product) that captures and visualizes all the data your team needs in one place. Complete with product analytics, feature flags, session recording, heatmaps, funnels and more, it’s a great choice if you want to build a comprehensive, data-driven product strategy, fast.
```

Comment:
Do we want to send them to `app.posthog.com/signup` from here instead?

Reply:
No, it's not a natural journey to go direct to the signup page.